app/multitenant/dynamo_collector.go

```diff
@@ -138,9 +126,10 @@ type watchKey struct {
 // https://github.com/aws/aws-sdk-go/wiki/common-examples
 func NewDynamoDBCollector(
 	userIDer UserIDer,
-	dynamoDBConfig, s3Config *aws.Config,
-	tableName, bucketName, natsHost, memcachedHost string,
-	memcachedTimeout time.Duration, memcachedService string,
+	dynamoDBConfig *aws.Config, tableName string,
+	s3Store *S3Store,
+	natsHost string,
+	memcacheClient *MemcacheClient,
```

Comment:
We discussed having a config struct in the previous PR.  Now seems like a good opportunity.